saviSimulation/savi/commandStation/CommandStationCore.java

```diff
@@ -96,9 +89,10 @@ public void receiveMessage(String msg) {
 			parsedMessage = item.toString();
 			//parsedMessage = "";
 		}
-		
+
 		// Send to the GUI
 		gui.receiveMessage(msg, parsedMessage);
 	}
+
 }
 
```

Comment:
public static void main(String[] args) {
    	new CommandStationGUI("commander");
}

Reply:
I've resolved this issue.